### PR TITLE
Added Archive, Trash and Undo Function

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Screenshot (1039)
Screenshot (1038)

### CODE REVIEW

**Review by mnath30 — APPROVED** [review]
Code is well structured and ui looks great

File: src/Components/NotesList/AddNote.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    deleteNote(perticularNotes._id); // API call
+
+    setTrashNotes([...trashNotes, perticularNotes]); // ADDING to TRASH
+
+    // DELETING NOTE
+    // setNotes(notes.filter((note) => note._id !== id));
+
+    // function moveToTrash(id) {
+    //   const tempNotes = [...notes];
+    //   const index = tempNotes.findIndex((item) => item.id === id);
```
Comment (srishtigautam21) [inline src/Components/NotesList/AddNote.js:29]: Remove commented code

File: src/Components/NotesList/NotesList.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/* .notes__list_container {
+  margin: 1rem;
```
Comment (srishtigautam21) [inline src/Components/NotesList/NotesList.css:2]: commented code

**Review by ShrutiLakhani — APPROVED** [review]
Code looks fine. Well structured. Good job 👍 
Remove commented code, if not required.

File: src/Context/AuthContext.js [hunk 3]
```diff
@@ -11,18 +11,18 @@ const AuthContext = createContext();
...
-        console.log("Masla hgya ji USEEFFECT");
-      }
-    }
-    oneTime(authDispatch);
-  }, []);
+  // useEffect(() => {
+  //   function oneTime(authDispatch) {
+  //     console.log("I am here in USEEFFECT");
+  //     if (localStorage.getItem("token" !== null)) {
+  //       console.log("Login is continuing");
```
Comment (srishtigautam21) [inline src/Context/AuthContext.js:18]: commented code

**Review by srishtigautam21 — APPROVED** [review]
The code is clean
Looks good to merge with changes

File: src/Context/AuthContext.js (hunk 3 above)
Comment (ashishkatiyar2404) [reply src/Context/AuthContext.js:18]: will remove thanks

**Comment by ashishkatiyar2404** [comment]
> Code looks fine. Well structured. Good job 👍 Remove commented code, if not required.

ok

**Pull request merged**